### PR TITLE
feat: fetch source as listed from cdn

### PR DESCRIPTION
**Pull request by cangzhang**

### CODE REVIEW

File: src/modules/import/index.tsx [hunk 1]
```diff
@@ -13,17 +13,16 @@ import { useStyletron } from 'baseui';
 import { toaster, ToasterContainer, PLACEMENT } from 'baseui/toast';
 import { Button } from 'baseui/button';
 
-import Sources from 'src/share/constants/sources';
+import Sources, { PkgList } from 'src/share/constants/sources';
```
Comment (uniquemo) [inline src/modules/import/index.tsx:16]: test

Comment (uniquemo) [reply src/modules/import/index.tsx:16]: abc

File: src/modules/popup/index.tsx [hunk 2]
```diff
@@ -19,11 +19,11 @@ import { Popover, StatefulPopover, TRIGGER_TYPE } from 'baseui/popover';
 import config from 'src/native/config';
 import { QQChampionAvatarPrefix } from 'src/share/constants/sources';
 import LCUService from 'src/service/lcu';
+import Sources, { PkgList, SourceList } from 'src/share/constants/sources';
```
Comment (uniquemo) [inline src/modules/popup/index.tsx:22]: aaa

**Review by uniquemo — CHANGES_REQUESTED** [review]
111

File: src/modules/import/index.tsx [hunk 3]
```diff
@@ -13,17 +13,16 @@ import { useStyletron } from 'baseui';
...
 import {
   prepareReimport,
   updateFetchingSource,
   importBuildFailed,
   importBuildSucceed,
 } from 'src/share/actions';
 import { removeFolderContent } from 'src/share/file';
```
Comment (uniquemo) [inline src/modules/import/index.tsx:23]: 123

**Review by zhouqiui — APPROVED** [review]
test

**Review by zhouqiui — CHANGES_REQUESTED** [review]
222

File: src/modules/home/index.tsx [hunk 4]
```diff
@@ -20,12 +20,11 @@ import { H6 } from 'baseui/typography';
 import config from 'src/native/config';
 import { updateConfig, updateDataSourceVersion } from 'src/share/actions';
 import { ChampionKeys } from 'src/share/constants/champions';
-import Sources, { isAram } from 'src/share/constants/sources';
+import Sources, { PkgList, SourceList } from 'src/share/constants/sources';
 import AppContext from 'src/share/context';
-import OpGG from 'src/service/data-source/op-gg';
-import LolQQ from 'src/service/data-source/lol-qq';
```
Comment (zhouqiui) [inline src/modules/home/index.tsx:26]: 12344

**Review by zhouqiui — APPROVED** [review]

File: src/modules/home/index.tsx [hunk 5]
```diff
@@ -20,12 +20,11 @@ import { H6 } from 'baseui/typography';
 import config from 'src/native/config';
 import { updateConfig, updateDataSourceVersion } from 'src/share/actions';
 import { ChampionKeys } from 'src/share/constants/champions';
-import Sources, { isAram } from 'src/share/constants/sources';
+import Sources, { PkgList, SourceList } from 'src/share/constants/sources';
 import AppContext from 'src/share/context';
-import OpGG from 'src/service/data-source/op-gg';
```
Comment (zhouqiui) [inline src/modules/home/index.tsx:25]: 333

Comment (zhouqiui) [inline src/modules/home/index.tsx:25]: 434

**Review by zhouqiui — CHANGES_REQUESTED** [review]

**Review by zhouqiui — APPROVED** [review]

**Pull request merged**